### ISSUE TITLE
Add support for java.util.Map

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vadbch/orika#15]
```
Currently, Orika has no built-in support for java.util.Map in it's mapping 
methods or auto-generated mappings by configuration.

This should be implemented as part of the 1.2.0 release.
```

Original issue reported on code.google.com by `matt.deb...@gmail.com` on 9 May 2012 at 11:40

**Comment by GoogleCodeExporter** [comment]
```
In the 1.2.0 branch:
Added initial support to MapperFacade with methods to support maps: mapAsMap, 
mapAsList, mapAsSet, mapAsArray permutations, along with an initial set of test 
cases. 

Still need map handling for ClassMapBuilder and configuration-generated mappers.
```

Original comment by `matt.deb...@gmail.com` on 10 May 2012 at 12:22

**Comment by GoogleCodeExporter** [comment]
```
Support for configuration-generated mappers is also available in 1.2.0 branch.
```

Original comment by `matt.deb...@gmail.com` on 11 Jul 2012 at 6:23
- Changed state: **Fixed**
